op-e2e/system/altda/concurrent_test.go

```diff
@@ -55,23 +55,20 @@ func TestBatcherConcurrentAltDARequests(t *testing.T) {
 	err = driver.StartBatchSubmitting()
 	require.NoError(t, err)
 
-	totalBatcherTxsCount := int64(0)
-	// wait for up to 5 L1 blocks, expecting 10 L2 batcher txs in them.
-	// usually only 3 is required, but it's possible additional L1 blocks will be created
-	// before the batcher starts, so we wait additional blocks.
-	for i := int64(0); i < 5; i++ {
-		block, err := geth.WaitForBlock(big.NewInt(int64(startingL1BlockNum)+i), l1Client, time.Duration(cfg.DeployConfig.L1BlockTime*2)*time.Second)
+	// Iterate over up to 10 blocks. The number of transactions sent by the batcher should
+	// exceed the number of blocks.
+	checkBlocks := 10
+	for i := 0; i < checkBlocks; i++ {
+		block, err := geth.WaitForBlock(big.NewInt(int64(startingL1BlockNum)+int64(i)), l1Client, time.Duration(cfg.DeployConfig.L1BlockTime*2)*time.Second)
 		require.NoError(t, err, "Waiting for l1 blocks")
 		// there are possibly other services (proposer/challenger) in the background sending txs
 		// so we only count the batcher txs
 		batcherTxCount, err := transactions.TransactionsBySender(block, cfg.DeployConfig.BatchSenderAddress)
 		require.NoError(t, err)
-		totalBatcherTxsCount += int64(batcherTxCount)
-
-		if totalBatcherTxsCount >= numL1TxsExpected {
+		if batcherTxCount > 1 {
```

Comment:
it used to be looking out for 10 transactions, but now it wants just one. Might we want a higher value than just 1?

Reply:
I modified the algorithm to check if there's at least 1 block with more than one batcher tx in it. The only way that can happen is if the tx manager is able to successfully submit transactions concurrently, since it waits for confirmations in serial mode.